tests/conllu_test.py

```diff
@@ -4,77 +4,197 @@
 import tempfile
 import unittest
 
-from udtube import data
+from udtube.data import conllu
+
+
+class IDTest(unittest.TestCase):
+
+    def test_swe(self):
+        swe = conllu.ID(3)
+        self.assertEqual(len(swe), 1)
+        self.assertFalse(swe.is_mwe)
+        self.assertEqual(str(swe), "3")
+
+    def test_swe_roundtrip(self):
+        swe = conllu.ID(3)
+        self.assertEqual(swe, conllu.ID.parse_from_string(str(swe)))
+
+    def test_mwe(self):
+        mwe = conllu.ID(2, 3)
+        self.assertEqual(len(mwe), 2)
+        self.assertTrue(mwe.is_mwe)
+        self.assertEqual(str(mwe), "2-3")
+
+    def test_mwe_roundtrip(self):
+        mwe = conllu.ID(2, 3)
+        self.assertEqual(mwe, conllu.ID.parse_from_string(str(mwe)))
+
+
```

Comment:
I would add tests comparing 
1. two MWE to each other (the __eq__ stuff)
2. an MWE to a SWE with __eq__)
3. Very optional, but a failed parse (2:3, or something)

Reply:
```suggestion

 def test_mwe_non_equivalence(self):
        mwe1 = conllu.ID(2,3)
        mwe2 = conllu.ID(3,4)
        self.assertNotEqual(mwe1, mwe2)

    def test_mwe_swe_equivalence(self):
        mwe = conllu.ID(2, 3)
        swe = conllu.ID(2)
        self.assertNotEqual(mwe, swe)

    def test_bad_parse(self):
        with self.assertRaises(Error):
            bad_mwe = conllu.ID.parse_from_string("2:3")
```

Reply:
These are good tests and I appreciate putting them into code. Merged in.